dask/base.py

```diff
@@ -816,3 +799,64 @@ def _colorize(t):
     h = hex(int(i))[2:].upper()
     h = '0' * (6 - len(h)) + h
     return "#" + h
+
+
+named_schedulers = {
+    'sync': local.get_sync,
+    'synchronous': local.get_sync,
+    'single-threaded': local.get_sync,
+    'threads': threaded.get,
+    'threaded': threaded.get,
+}
+
+try:
+    from dask import multiprocessing as dask_multiprocessing
+except ImportError:
+    pass
+else:
+    named_schedulers.update({
+        'processes': dask_multiprocessing.get,
+        'multiprocessing': dask_multiprocessing.get,
+    })
+
+
+def get_scheduler(get=None, scheduler=None, collections=None, cls=None):
+    if get is not None:
+        if scheduler is not None:
+            raise ValueError("Both get= and scheduler= provided.  Choose one")
+        return get
+
+    if scheduler is not None:
+        if scheduler.lower() in named_schedulers:
+            return named_schedulers[scheduler.lower()]
+        elif scheduler.lower() in ('dask.distributed', 'distributed'):
+            from distributed.worker import get_client
+            return get_client().get
```

Comment:
What if this isn't running in a worker? Error nicely? Where would you expect this to come up?

Reply:
This will work outside of a worker if a client has been made in the process.  

If no client has been made and we're not in a worker then they'll get this error

```python
In [2]: get_client()
ValueError: No global client found and no address provided
```